src/system-health/health_checker/config.py

```diff
@@ -20,7 +20,7 @@ class Config(object):
     DEFAULT_LED_CONFIG = {
         'fault': 'red',
         'normal': 'green',
-        'booting': 'orange_blink'
+        'booting': 'red'
```

Comment:
why changed from 'orange' blinking to 'red'? How to distinguish between fault state or system hung stat during boot?

Reply:
This is for backward compatible. Before this change, booting stage is not used, so "orange_blink" is not used too, system led color is always red during system booting time because not all services are ready. In the new change, I set "booting stage" to red to make sure backward compatible: if user does not provide system health configuration file, system led color will be red which is the same behavior as before; otherwise, system led will be set to the color specified in system health configuration file.